site/content/en/docs/tasks/run/deployment.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Run Deployment"
+linkTitle: "Deployment"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a Deployment as a Kueue-managed job.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running Deployments.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
```

Comment:
I would suggest to introduce "Serving user", with the description. As a description maybe we can have 
"A serving user runs serving workloads, for example, to expose a trained AI/ML model for inference"